features/shared-memory.yml

```diff
@@ -21,7 +24,7 @@ compat_features:
   - javascript.builtins.Atomics.xor
   - javascript.builtins.DataView.DataView.sharedarraybuffer_support
   - javascript.builtins.SharedArrayBuffer
+  - javascript.builtins.SharedArrayBuffer.@@species
```

Comment:
For posterity, see also: https://github.com/web-platform-dx/web-features/pull/2268#issuecomment-2483369844